File: modul_praktikum_python/contoh_code/pascal/cek_tombol_a.pas
uses crt; 
var 
  A : char; 
begin 
  write(‘Masukkan Suatu Karakter :’); 
  A:=readkey; 
  writeln; 
  if A=’A’ then  
     writeln(‘Anda menekan A besar’) 
  else 
     writeln(‘Anda tidak menekan A besar’); 
  readkey; 
end.

File: modul_praktikum_python/contoh_code/python/cek_tombol_a.py
"""Cek apakah pengguna menekan huruf A besar."""


def main() -> None:
    s = input("Masukkan Suatu Karakter :").strip()
    a = s[0] if s else ""
    print()
    if a == "A":
        print("Anda menekan A besar")
    else:
        print("Anda tidak menekan A besar")
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()
